tutorials/navigation/navigation_using_navigationagents.rst

```diff
@@ -3,6 +3,10 @@
 Using NavigationAgents
 ======================
 
+.. tip::
+
+    For more advanced uses consider :ref:`doc_navigation_using_navigationpathqueryobjects` over NavigationAgent nodes.
```

Comment:
Feels like this tip is better suited to the bottom of this section.
It's not even begun talking about navigation agents and the page is already suggesting to rethink your choice

Reply:
Agreed